### PR TITLE
XW-1077 | Upgrade to Ember 2.1

### PR DESCRIPTION
**Pull request by bognix**
## Links

* http://wikia-inc.atlassian.net/browse/XW-1077

## Description

Update Ember from 2.0.3  to 2.1

## Reviewers

@Wikia/x-wing

### CODE REVIEW

File: front/main/tests/integration/components/infobox-builder-test.js [hunk 1]
```diff
@@ -8,10 +8,15 @@ moduleForComponent('infobox-builder', 'Integration | Component | infobox builder
 
 test('reset item in edit mode on clicking preview background', function (assert) {
 	const setEditItemSpy = sinon.spy(),
+		removeItemMock = () => {},
```
Comment (vforge) [inline front/main/tests/integration/components/infobox-builder-test.js]: There's no need for this variable (or one above)

**Comment by vforge** [comment]
LGTM

File: front/main/app/components/article-content.js [hunk 2]
```diff
@@ -354,15 +354,21 @@ export default Ember.Component.extend(
...
+
+			component.setProperties({
+				ref,
+				width: parseInt(element.getAttribute('width'), 10),
+				height: parseInt(element.getAttribute('height'), 10),
+				imgWidth: element.offsetWidth,
+				media
+			});
+
+			view = this.createChildView(component).createElement();
```
Comment (rogatty) [inline front/main/app/components/article-content.js]: What do you think about naming it `componentElement`?

**Comment by rogatty** [comment]
LGTM

**Pull request merged**